src/libertem/web/cli.py

```diff
@@ -7,18 +7,18 @@
               default=9000, type=int)
 @click.option('--local-directory', help='local directory to manage dask-worker-space files',
               default='dask-worker-space', type=str)
-@click.option('--no-browser', help='this flag disables opening the browser', is_flag=True)
+@click.option('--browser/--no-browser', help='this flag enable/disable opening the browser', default='True')
```

Comment:
This is the line mentioned in the flake8 report. I would suggest dropping the "this flag " prefix, which is kind of a 'filler' here anyways.

Reply:
Got it. Can I run those test locally ?

Reply:
Yes, with `tox -e flake8`. Or even better, activate flake8 checks in the editor if you are using one that supports this. That's faster and more convenient IMO.